diff --git a/dist/index.js b/dist/index.js
index dc37ac4a018a086c4244a09a67215dbaa9b4de65..e51d01138aa56ebebac6ef0dc6d2ec48252ec621 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -179,7 +179,7 @@ var Select = (props) => {
             children
           }
         ) }),
-        isFormControl ? /* @__PURE__ */ (0, import_jsx_runtime.jsxs)(
+        (isFormControl && nativeOptionsSet.size > 0) ? /* @__PURE__ */ (0, import_jsx_runtime.jsxs(
           SelectBubbleInput,
           {
             "aria-hidden": true,
diff --git a/dist/index.mjs b/dist/index.mjs
index f9b94f39dfddef678ef3086354f8d7413ae27e52..f6f1c165ff0b07eccd817f0e359101d611ea0a1f 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -112,7 +112,7 @@ var Select = (props) => {
             children
           }
         ) }),
-        isFormControl ? /* @__PURE__ */ jsxs(
+        (isFormControl && nativeOptionsSet.size > 0) ? /* @__PURE__ */ jsxs(
           SelectBubbleInput,
           {
             "aria-hidden": true,
